modules/product/src/main/java/com/opengamma/strata/product/SecurityInfoBuilder.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.joda.beans.PropertyDefinition;
+
+/**
+ * Builder to create {@code SecurityInfo}.
+ * <p>
+ * This builder allows a {@link SecurityInfo} to be created.
+ */
+public final class SecurityInfoBuilder {
+
+  /**
+   * The security identifier.
+   * <p>
+   * This identifier uniquely identifies the security within the system.
+   */
+  private SecurityId id;
+  /**
+   * The information about the security price.
+   * <p>
+   * This provides information about the security price.
+   * This can be used to convert the price into a monetary value.
+   */
+  @PropertyDefinition(validate = "notNull")
```

Comment:
This annotation doesn't need to be here. Presumably a leftover from pasting the fields from `SecurityInfo`